Feature 361 install instruct (#363)
dtcenter/METdatadb: docs/Users_Guide/installation.rst

  git clone https://github.com/dtcenter/METdataio

> Instructions for installing the metdataio package locally

Reviewer: ```suggestion
Instructions for installing the METdataio package locally via conda
```

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

>  * Activate the conda environment.  

Reviewer: ```suggestion
 * Activate the conda environment, changing <your-conda-env-name> to the correct location.
```

>    Please change 'your-conda-env-name' to the correct location.

Reviewer: ```suggestion
```

    .. code-block:: ini

>       conda activate your-conda-env-name

Reviewer: ```suggestion
      conda activate <your-conda-env-name>
```

>  * From within the active conda environment, cd to the *METdataio/* directory,

Reviewer: ```suggestion
 * From within the active conda environment, change directories to be in the *METdataio/* directory,
```

   where the file **pyproject.toml** is located.

 * From this directory, run the following on the command line:
   
   .. code-block:: ini

      pip install -e .

>  * The -e option stands for editable, which is useful to

Reviewer: ```suggestion
 * The `-e` option stands for editable, which is useful to
```

>  * The `-e` option stands for editable, which is useful to

Reviewer: ```suggestion
 * The :code:`-e` option stands for editable, which is useful to
```

   update the user's METdataio source without reinstalling it.

>  * The . indicates that the user should search the current directory for

Reviewer: ```suggestion
 * The `.` indicates that the user should search the current directory for
```

>  * The `.` indicates that the user should search the current directory for

Reviewer: ```suggestion
 * The :code:`.` indicates that the user should search the current directory for
```

   the **pyproject.toml** file.

>  * Use metdataio package via import statement:

Reviewer: ```suggestion
 * To use the modules in the *METdataio/METdbLoad/ush* directory, use the metdataio package via an import statement:
```

>     * From metdataio import **metdbload.ush**

Reviewer: ```suggestion
   .. code-block:: ini
      from metdataio import metdbload.ush
```

Reviewer: @bikegeek Can you please review this suggested change to confirm what I am suggesting is accurate and makes sense?

>         * To use the modules in the *METdataio/METdbLoad/ush* directory.

Reviewer: ```suggestion
```
